apiml-security-common/README.md

```diff
@@ -0,0 +1,3 @@
+# Security Service Client library
+
+The `security-service-client-spring` enables authentication and protection using security providers. Follow [this link](https://zowe.github.io/docs-site/latest/extend/extend-apiml/api-mediation-security.html#security-service-client-library) to get more information about this library and its components.
```

Comment:
Replace "Follow [this link](https://zowe.github.io/docs-site/latest/extend/extend-apiml/api-mediation-security.html#security-service-client-library) to get more information about this library and its components." with 
"**For more information, see [Security Service Client Spring](https://zowe.github.io/docs-site/latest/extend/extend-apiml/api-mediation-security.html#security-service-client-library)**.

Reply:
done